src/main/java/com/minecolonies/coremod/util/InventoryUtils.java

```diff
@@ -448,7 +448,7 @@ public static int getFirstSlotContainingTool(@NotNull final IInventory inventory
             final ItemStack item = inventory.getStackInSlot(i);
             //Only classic fishingRod recognized as a fishingTool
             if (item != null && (item.getItem().getToolClasses(item).contains(tool) || ("hoe".equals(tool) && item.getUnlocalizedName().contains("hoe"))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (5) used in the expression (maximum allowed 4). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1067)